editoast/src/views/path/projection.rs

```diff
@@ -437,20 +437,68 @@ mod tests {
         #[case] path: &[&str],
         #[case] track_ranges: &[&str],
         #[case] expected_intersections: &[(u64, u64)],
+        #[values(false, true)] toggle_path: bool,
+        #[values(false, true)] toggle_track_ranges: bool,
     ) {
-        let path: Vec<TrackRange> = path.into_iter().map(|s| s.parse().unwrap()).collect();
```

Comment:
Just a reminder from our call: some of the code here could be moved into a separate function for reuse.